chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java

```diff
@@ -16,33 +16,71 @@
     private final long requestsPerSecond;
     private final long maxLoadThreads;
     private final long warmUpTimeInSeconds;
+    private final int tickInterval;
 
-    /** Create load profile: request per seconds
-     * @n
-     * @param requestsPerSecond - The number of requests per second Jagger shall perform
-     * @param maxLoadThreads - The maximum number of threads, which Jagger engine can create to provide the requested load
-     * @param warmUpTimeInSeconds - The warm up time value in seconds. Jagger increases load from 0 to @b requestPerSecond by @b warmUpTimeInSeconds
-     */
-    public JLoadProfileRps(RequestsPerSecond requestsPerSecond, MaxLoadThreads maxLoadThreads, WarmUpTimeInSeconds warmUpTimeInSeconds) {
-        Objects.nonNull(requestsPerSecond);
-        Objects.nonNull(maxLoadThreads);
-        Objects.nonNull(warmUpTimeInSeconds);
-        
-        this.requestsPerSecond = requestsPerSecond.value();
-        this.maxLoadThreads = maxLoadThreads.value();
-        this.warmUpTimeInSeconds = warmUpTimeInSeconds.value();
+    private JLoadProfileRps(Builder builder) {
+        Objects.nonNull(builder);
+
+        this.requestsPerSecond = builder.requestsPerSecond;
+        this.maxLoadThreads = builder.maxLoadThreads;
+        this.warmUpTimeInSeconds = builder.warmUpTimeInSeconds;
+        this.tickInterval = builder.tickInterval;
     }
 
-    /** Create load profile: request per seconds
+    /** Builder of the JLoadProfileRps: request per seconds
      * @n
-     * @param requestsPerSecond - The number of requests per second Jagger shall perform
-     * @param maxLoadThreads - The maximum number of threads, which Jagger engine can create to provide the requested load
+     * @details Constructor parameters are mandatory for the JLoadProfileRps. All parameters, set by setters are optional
+     * @n
+     * @param requestsPerSecond   - The number of requests per second Jagger shall perform
+     * @param maxLoadThreads      - The maximum number of threads, which Jagger engine can create to provide the requested load
      * @param warmUpTimeInSeconds - The warm up time value in seconds. Jagger increases load from 0 to @b requestPerSecond by @b warmUpTimeInSeconds
      */
-    public static JLoadProfileRps of(RequestsPerSecond requestsPerSecond, MaxLoadThreads maxLoadThreads, WarmUpTimeInSeconds warmUpTimeInSeconds) {
-        return new JLoadProfileRps(requestsPerSecond, maxLoadThreads, warmUpTimeInSeconds);
+    public static Builder builder(RequestsPerSecond requestsPerSecond, MaxLoadThreads maxLoadThreads, WarmUpTimeInSeconds warmUpTimeInSeconds) {
```

Comment:
maxloadthreads & warmuptime are also oprional

see: com.griddynamics.jagger.xml.beanParsers.task.TpsDefinitionParser
for previous usage and default values